PhpAmqpLib/Channel/AMQPChannel.php

```diff
@@ -844,13 +844,19 @@ public function basic_nack($delivery_tag, $multiple = false, $requeue = false)
      *
      * @param string $consumer_tag
      * @param bool $nowait
+     * @param bool $noreturn
      * @return mixed
      */
-    public function basic_cancel($consumer_tag, $nowait = false)
+    public function basic_cancel($consumer_tag, $nowait = false, $noreturn = false)
```

Comment:
As I said earlier, `$noreturn` is not part of the spec, and it seems to duplicate what `$nowait` is made for